assets/js/windup.js

```diff
@@ -1,5 +1,5 @@
 $(".windup-download-now").click(function() {
-    window.location = 'https://www.jboss.org/download-manager/file/jboss-migrationtoolkit-windup-2.4.0.GA.zip'
+    window.location = 'https://www.jboss.org/download-manager/file/windup-distribution-2.5.0.Final-offline.zip'
```

Comment:
The link doesn't work for me:
https://www.jboss.org/download-manager/file/windup-distribution-2.5.0.Final-offline.zip